SofaKernel/framework/sofa/helper/vector.cpp

```diff
@@ -51,6 +52,62 @@ void SOFA_HELPER_API vector_access_failure(const void* vec, unsigned size, unsig
     assert(i < size);
 }
 
-} // namespace helper
+/// Convert the string 's' into an unsigned int. The error are reported in msg & numErrors
+/// is incremented.
+int SOFA_HELPER_API getInteger(const std::string& s, std::stringstream& msg, unsigned int& numErrors)
+{
+    const char* attrstr=s.c_str();
```

Comment:
Ok this is likely due to copying of the `std::string` in the `std::stringstream` constructor. The right approach would be to work directly from the input file stream anyways, which *should* not be any slower. 

I might give it a quick shot, where are the benchmarks located?